scripts/biomedical/ENCODE/schema.py

```diff
@@ -0,0 +1,64 @@
+"""Script for consolidating ENCODE vocab.
+
+Flume job that consolidates all new schema vocab generated by the ENCODE
+project.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import google_type_annotations
+from __future__ import print_function
+
+import logging
+import google3
+
+import apache_beam as beam
+
+from absl import app
+from absl import flags
+
+from google3.pipeline.flume.py import runner
+
+# Logger
+LOGGER = logging.getLogger('encode_scraper')
+
+# Where all schema files are stored
+INPUT_PATTERN = '/cns/jv-d/home/datcom/encode_project/schema/*'
```

Comment:
Replace these paths with '/tmp/encode/schema/*' and '/tmp/encode/final_schema.txt' or something ?

Reply:
done